verify/verify.go

```diff
@@ -89,6 +89,11 @@ func Files(ps client.PackageState) (bool, error) {
 		if err != nil {
 			return false, err
 		}
+		fstat, _ := os.Stat(file)
+		// Only calculate/ check the checksum for file, not folder.
```

Comment:
You will want to check error here to avoid edge cases

Reply:
Thank you. Added error checking for the `os.Stat`.